static/templates/left_sidebar.hbs

```diff
@@ -1,5 +1,10 @@
 <div class="left-sidebar" id="left-sidebar" role="navigation">
     <div class="narrows_panel">
+        {{#if display_organization_name}}
+        <div class= "company-name">
+            {{realm_name}}
+        </div>
+        {{/if}}
```

Comment:
I think you want a persistent element that is hidden-or-not, so that we don't need to rerender the template to live update after the setting is changed.

Also, the name `company-name` isn't appropriate; since that's not what we call that in zulip and many users are not companies.